.github/workflows/deploy.yml

```diff
@@ -59,6 +57,13 @@ jobs:
       - name: build
         run: yarn build
 
+
+      # upload coverage
+      - name: Coveralls
```

Comment:
Seems like this has to be run immediately after running tests (it currently [looks](https://github.com/exoframejs/exoframe-server/runs/534749863) for lcov file and fails to find one)